agent/build.gradle

```diff
@@ -51,7 +51,10 @@ shadowJar {
     relocate('org.slf4j', "${shadowPrefix}.org.slf4j")
     // FIXME need to relocate logback also, but currently causes instrumentation to be relocated
     // relocate('ch.qos.logback', "${shadowPrefix}.ch.qos.logback")
-    relocate('com.fasterxml.jackson', "${shadowPrefix}.com.fasterxml.jackson")
+
+    // this shades internal use of java util logging which is needed to prevent java util logging from being initialized
+    // early (e.g. in the case of jboss which wants to do some setup before jul is initialized)
+    relocate('java.util.logging', 'org.glowroot.xyzzy.engine.jul')
```

Comment:
@trask, Do you think this will fix the issue with JBoss in master?

Reply:
It should fix the issue with java.util.logging

Reply:
I'm going to give it a try